ntoskrnl/mm/section.c

```diff
@@ -169,6 +201,51 @@ static GENERIC_MAPPING MmpSectionMapping =
 /* FUNCTIONS *****************************************************************/
 
 
+
+NTSTATUS
+NTAPI
+MiWritePage(PMM_SECTION_SEGMENT Segment,
```

Comment:
The goal is to have Cc & Mm separated enough so that further work on Mm is possible. I tried doing it the other way, and believe me, it's a mess. Now the work of keeping the state of dirty pages for data section & flushing them to disk is on Mm side. You need new functions for this.

If the question is: "Why not adding them to the new Mm & forward to the old one ?", the answer is "what's the point ? This will need a rewrite to switch to the new Mm anyway"